_episodes_rmd/07-control-flow.Rmd

```diff
@@ -59,15 +59,22 @@ if (x >= 10) {
 x
 ```
 
-Note you may not get the same output as your neighbour because
-you may be sampling different random numbers from the same distribution.
+The print statement does not appear in the console because x is not greater than 10. To print a different message for numbers less than 10, we can add an `else` statement.
 
-Let's set a seed so that we all generate the same 'pseudo-random'
-number, and then print more information:
+```{r}
+x <- 8
+
+if (x >= 10) {
+  print("x is greater than or equal to 10")
+} else {
+  print("x is less than 10")
+}
+```
+
+You can also test multiple conditions by using `else if`.
 
 ```{r}
-set.seed(10)
-x <- rpois(1, lambda=8)
+x <- 8
 
 if (x >= 10) {
   print("x is greater than or equal to 10")
```

Comment:
I know this is not part of your PR, but can you also modify the `else if` statement here, so that it reads `print("x is greater than 5 and less than 10")`.